app/Models/Mship/Account.php

```diff
@@ -460,6 +462,17 @@ public function isDuplicateDisplayName($displayName)
         })->isEmpty();
     }
 
+    /**
+     * Returns the Discord user associated with this account if the user has linked it.
+     *
+     * @return RestCord/Model/User/User The Discord user
+     */
+    public function getDiscordUserAttribute()
+    {
+        $discord = app()->make(Discord::class);
+        return $discord->getUserInformation($this);
```

Comment:
```suggestion
        return app()->make(Discord::class)->getUserInformation($this);
```
This should work and helps clean it up by removing a temp var.

Reply:
Also a good catch! https://github.com/VATSIM-UK/core/pull/2029/commits/bacf74390e917a543d3d37b3513e289f79bdac1a